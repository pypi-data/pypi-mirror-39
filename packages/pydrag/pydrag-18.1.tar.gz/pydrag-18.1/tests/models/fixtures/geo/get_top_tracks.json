{
    "interactions": [
        {
            "request": {
                "body": null,
                "headers": {
                    "Accept": [
                        "*/*"
                    ],
                    "Accept-Encoding": [
                        "gzip, deflate"
                    ],
                    "Connection": [
                        "keep-alive"
                    ],
                    "User-Agent": [
                        "python-requests/2.20.0"
                    ]
                },
                "method": "GET",
                "uri": "https://ws.audioscrobbler.com/2.0/?api_key=LAST_FM_API_KEY&country=greece&format=json&limit=10&method=geo.getTopTracks&page=1"
            },
            "response": {
                "body": {
                    "string": "{\"tracks\":{\"track\":[{\"name\":\"Do I Wanna Know?\",\"duration\":\"272\",\"listeners\":\"867724\",\"mbid\":\"CENSORED1a11\",\"url\":\"https://www.last.fm/music/Arctic+Monkeys/_/Do+I+Wanna+Know%3F\",\"streamable\":{\"#text\":\"0\",\"fulltrack\":\"0\"},\"artist\":{\"name\":\"Arctic Monkeys\",\"mbid\":\"CENSORED298a\",\"url\":\"https://www.last.fm/music/Arctic+Monkeys\"},\"image\":[{\"#text\":\"https://lastfm-img2.akamaized.net/i/u/34s/CENSORED.png\",\"size\":\"small\"},{\"#text\":\"https://lastfm-img2.akamaized.net/i/u/64s/CENSORED.png\",\"size\":\"medium\"},{\"#text\":\"https://lastfm-img2.akamaized.net/i/u/174s/CENSORED.png\",\"size\":\"large\"},{\"#text\":\"https://lastfm-img2.akamaized.net/i/u/300x300/CENSORED.png\",\"size\":\"extralarge\"}],\"@attr\":{\"rank\":\"0\"}},{\"name\":\"Burn the Witch\",\"duration\":\"0\",\"listeners\":\"261777\",\"mbid\":\"\",\"url\":\"https://www.last.fm/music/Radiohead/_/Burn+the+Witch\",\"streamable\":{\"#text\":\"0\",\"fulltrack\":\"0\"},\"artist\":{\"name\":\"Radiohead\",\"mbid\":\"CENSORED2711\",\"url\":\"https://www.last.fm/music/Radiohead\"},\"image\":[{\"#text\":\"https://lastfm-img2.akamaized.net/i/u/34s/CENSORED.png\",\"size\":\"small\"},{\"#text\":\"https://lastfm-img2.akamaized.net/i/u/64s/CENSORED.png\",\"size\":\"medium\"},{\"#text\":\"https://lastfm-img2.akamaized.net/i/u/174s/CENSORED.png\",\"size\":\"large\"},{\"#text\":\"https://lastfm-img2.akamaized.net/i/u/300x300/CENSORED.png\",\"size\":\"extralarge\"}],\"@attr\":{\"rank\":\"1\"}},{\"name\":\"Teardrop\",\"duration\":\"279\",\"listeners\":\"1288368\",\"mbid\":\"CENSORED13ea\",\"url\":\"https://www.last.fm/music/Massive+Attack/_/Teardrop\",\"streamable\":{\"#text\":\"0\",\"fulltrack\":\"0\"},\"artist\":{\"name\":\"Massive Attack\",\"mbid\":\"CENSORED6ca8\",\"url\":\"https://www.last.fm/music/Massive+Attack\"},\"image\":[{\"#text\":\"https://lastfm-img2.akamaized.net/i/u/34s/CENSORED.png\",\"size\":\"small\"},{\"#text\":\"https://lastfm-img2.akamaized.net/i/u/64s/CENSORED.png\",\"size\":\"medium\"},{\"#text\":\"https://lastfm-img2.akamaized.net/i/u/174s/CENSORED.png\",\"size\":\"large\"},{\"#text\":\"https://lastfm-img2.akamaized.net/i/u/300x300/CENSORED.png\",\"size\":\"extralarge\"}],\"@attr\":{\"rank\":\"2\"}},{\"name\":\"Karma Police\",\"duration\":\"0\",\"listeners\":\"1605374\",\"mbid\":\"CENSORED927f\",\"url\":\"https://www.last.fm/music/Radiohead/_/Karma+Police\",\"streamable\":{\"#text\":\"0\",\"fulltrack\":\"0\"},\"artist\":{\"name\":\"Radiohead\",\"mbid\":\"CENSORED2711\",\"url\":\"https://www.last.fm/music/Radiohead\"},\"image\":[{\"#text\":\"https://lastfm-img2.akamaized.net/i/u/34s/CENSORED.png\",\"size\":\"small\"},{\"#text\":\"https://lastfm-img2.akamaized.net/i/u/64s/CENSORED.png\",\"size\":\"medium\"},{\"#text\":\"https://lastfm-img2.akamaized.net/i/u/174s/CENSORED.png\",\"size\":\"large\"},{\"#text\":\"https://lastfm-img2.akamaized.net/i/u/300x300/CENSORED.png\",\"size\":\"extralarge\"}],\"@attr\":{\"rank\":\"3\"}},{\"name\":\"Creep\",\"duration\":\"239\",\"listeners\":\"1607794\",\"mbid\":\"CENSOREDeaa6\",\"url\":\"https://www.last.fm/music/Radiohead/_/Creep\",\"streamable\":{\"#text\":\"0\",\"fulltrack\":\"0\"},\"artist\":{\"name\":\"Radiohead\",\"mbid\":\"CENSORED2711\",\"url\":\"https://www.last.fm/music/Radiohead\"},\"image\":[{\"#text\":\"https://lastfm-img2.akamaized.net/i/u/34s/CENSORED.png\",\"size\":\"small\"},{\"#text\":\"https://lastfm-img2.akamaized.net/i/u/64s/CENSORED.png\",\"size\":\"medium\"},{\"#text\":\"https://lastfm-img2.akamaized.net/i/u/174s/CENSORED.png\",\"size\":\"large\"},{\"#text\":\"https://lastfm-img2.akamaized.net/i/u/300x300/CENSORED.png\",\"size\":\"extralarge\"}],\"@attr\":{\"rank\":\"4\"}},{\"name\":\"Daydreaming\",\"duration\":\"0\",\"listeners\":\"215983\",\"mbid\":\"\",\"url\":\"https://www.last.fm/music/Radiohead/_/Daydreaming\",\"streamable\":{\"#text\":\"0\",\"fulltrack\":\"0\"},\"artist\":{\"name\":\"Radiohead\",\"mbid\":\"CENSORED2711\",\"url\":\"https://www.last.fm/music/Radiohead\"},\"image\":[{\"#text\":\"https://lastfm-img2.akamaized.net/i/u/34s/CENSORED.png\",\"size\":\"small\"},{\"#text\":\"https://lastfm-img2.akamaized.net/i/u/64s/CENSORED.png\",\"size\":\"medium\"},{\"#text\":\"https://lastfm-img2.akamaized.net/i/u/174s/CENSORED.png\",\"size\":\"large\"},{\"#text\":\"https://lastfm-img2.akamaized.net/i/u/300x300/CENSORED.png\",\"size\":\"extralarge\"}],\"@attr\":{\"rank\":\"5\"}},{\"name\":\"The Less I Know the Better\",\"duration\":\"0\",\"listeners\":\"382761\",\"mbid\":\"\",\"url\":\"https://www.last.fm/music/Tame+Impala/_/The+Less+I+Know+the+Better\",\"streamable\":{\"#text\":\"0\",\"fulltrack\":\"0\"},\"artist\":{\"name\":\"Tame Impala\",\"mbid\":\"CENSOREDef4d\",\"url\":\"https://www.last.fm/music/Tame+Impala\"},\"image\":[{\"#text\":\"https://lastfm-img2.akamaized.net/i/u/34s/CENSORED.png\",\"size\":\"small\"},{\"#text\":\"https://lastfm-img2.akamaized.net/i/u/64s/CENSORED.png\",\"size\":\"medium\"},{\"#text\":\"https://lastfm-img2.akamaized.net/i/u/174s/CENSORED.png\",\"size\":\"large\"},{\"#text\":\"https://lastfm-img2.akamaized.net/i/u/300x300/CENSORED.png\",\"size\":\"extralarge\"}],\"@attr\":{\"rank\":\"6\"}},{\"name\":\"Seven Nation Army\",\"duration\":\"449\",\"listeners\":\"1481075\",\"mbid\":\"CENSOREDabc5\",\"url\":\"https://www.last.fm/music/The+White+Stripes/_/Seven+Nation+Army\",\"streamable\":{\"#text\":\"0\",\"fulltrack\":\"0\"},\"artist\":{\"name\":\"The White Stripes\",\"mbid\":\"CENSOREDb3b5\",\"url\":\"https://www.last.fm/music/The+White+Stripes\"},\"image\":[{\"#text\":\"https://lastfm-img2.akamaized.net/i/u/34s/CENSORED.png\",\"size\":\"small\"},{\"#text\":\"https://lastfm-img2.akamaized.net/i/u/64s/CENSORED.png\",\"size\":\"medium\"},{\"#text\":\"https://lastfm-img2.akamaized.net/i/u/174s/CENSORED.png\",\"size\":\"large\"},{\"#text\":\"https://lastfm-img2.akamaized.net/i/u/300x300/CENSORED.png\",\"size\":\"extralarge\"}],\"@attr\":{\"rank\":\"7\"}},{\"name\":\"Come as You Are\",\"duration\":\"208\",\"listeners\":\"1789826\",\"mbid\":\"CENSORED004e\",\"url\":\"https://www.last.fm/music/Nirvana/_/Come+as+You+Are\",\"streamable\":{\"#text\":\"0\",\"fulltrack\":\"0\"},\"artist\":{\"name\":\"Nirvana\",\"mbid\":\"CENSOREDc4d6\",\"url\":\"https://www.last.fm/music/Nirvana\"},\"image\":[{\"#text\":\"https://lastfm-img2.akamaized.net/i/u/34s/CENSORED.png\",\"size\":\"small\"},{\"#text\":\"https://lastfm-img2.akamaized.net/i/u/64s/CENSORED.png\",\"size\":\"medium\"},{\"#text\":\"https://lastfm-img2.akamaized.net/i/u/174s/CENSORED.png\",\"size\":\"large\"},{\"#text\":\"https://lastfm-img2.akamaized.net/i/u/300x300/CENSORED.png\",\"size\":\"extralarge\"}],\"@attr\":{\"rank\":\"8\"}},{\"name\":\"Smells Like Teen Spirit\",\"duration\":\"301\",\"listeners\":\"2031860\",\"mbid\":\"CENSORED4ed7\",\"url\":\"https://www.last.fm/music/Nirvana/_/Smells+Like+Teen+Spirit\",\"streamable\":{\"#text\":\"0\",\"fulltrack\":\"0\"},\"artist\":{\"name\":\"Nirvana\",\"mbid\":\"CENSOREDc4d6\",\"url\":\"https://www.last.fm/music/Nirvana\"},\"image\":[{\"#text\":\"https://lastfm-img2.akamaized.net/i/u/34s/CENSORED.png\",\"size\":\"small\"},{\"#text\":\"https://lastfm-img2.akamaized.net/i/u/64s/CENSORED.png\",\"size\":\"medium\"},{\"#text\":\"https://lastfm-img2.akamaized.net/i/u/174s/CENSORED.png\",\"size\":\"large\"},{\"#text\":\"https://lastfm-img2.akamaized.net/i/u/300x300/CENSORED.png\",\"size\":\"extralarge\"}],\"@attr\":{\"rank\":\"9\"}}],\"@attr\":{\"country\":\"Greece\",\"page\":\"1\",\"perPage\":\"10\",\"totalPages\":\"252596\",\"total\":\"2525955\"}}}"
                },
                "headers": {
                    "Access-Control-Allow-Methods": [
                        "POST, GET, OPTIONS"
                    ],
                    "Access-Control-Allow-Origin": [
                        "*"
                    ],
                    "Access-Control-Max-Age": [
                        "86400"
                    ],
                    "Access-Control-Request-Headers": [
                        "Origin, X-Atmosphere-tracking-id, X-Atmosphere-Framework, X-Cache-Date, Content-Type, X-Atmosphere-Transport, *"
                    ],
                    "Connection": [
                        "keep-alive"
                    ],
                    "Content-Length": [
                        "8071"
                    ],
                    "Content-Type": [
                        "application/json; charset=UTF-8"
                    ],
                    "Date": [
                        "Sat, 10 Nov 2018 11:27:50 GMT"
                    ],
                    "Server": [
                        "openresty/1.9.7.5"
                    ]
                },
                "status": {
                    "code": 200,
                    "message": "OK"
                }
            }
        }
    ],
    "version": 1
}
