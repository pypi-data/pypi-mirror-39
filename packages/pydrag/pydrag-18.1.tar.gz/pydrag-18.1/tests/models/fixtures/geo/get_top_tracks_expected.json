{
    "country": "Greece",
    "data": [
        {
            "artist": {
                "mbid": "CENSORED298a",
                "name": "Arctic Monkeys",
                "url": "https://www.last.fm/music/Arctic+Monkeys"
            },
            "duration": 272,
            "image": [
                {
                    "size": "small",
                    "text": "https://lastfm-img2.akamaized.net/i/u/34s/CENSORED.png"
                },
                {
                    "size": "medium",
                    "text": "https://lastfm-img2.akamaized.net/i/u/64s/CENSORED.png"
                },
                {
                    "size": "large",
                    "text": "https://lastfm-img2.akamaized.net/i/u/174s/CENSORED.png"
                },
                {
                    "size": "extralarge",
                    "text": "https://lastfm-img2.akamaized.net/i/u/300x300/CENSORED.png"
                }
            ],
            "listeners": 867724,
            "mbid": "CENSORED1a11",
            "name": "Do I Wanna Know?",
            "rank": 0,
            "url": "https://www.last.fm/music/Arctic+Monkeys/_/Do+I+Wanna+Know%3F"
        },
        {
            "artist": {
                "mbid": "CENSORED2711",
                "name": "Radiohead",
                "url": "https://www.last.fm/music/Radiohead"
            },
            "duration": 0,
            "image": [
                {
                    "size": "small",
                    "text": "https://lastfm-img2.akamaized.net/i/u/34s/CENSORED.png"
                },
                {
                    "size": "medium",
                    "text": "https://lastfm-img2.akamaized.net/i/u/64s/CENSORED.png"
                },
                {
                    "size": "large",
                    "text": "https://lastfm-img2.akamaized.net/i/u/174s/CENSORED.png"
                },
                {
                    "size": "extralarge",
                    "text": "https://lastfm-img2.akamaized.net/i/u/300x300/CENSORED.png"
                }
            ],
            "listeners": 261777,
            "mbid": "",
            "name": "Burn the Witch",
            "rank": 1,
            "url": "https://www.last.fm/music/Radiohead/_/Burn+the+Witch"
        },
        {
            "artist": {
                "mbid": "CENSORED6ca8",
                "name": "Massive Attack",
                "url": "https://www.last.fm/music/Massive+Attack"
            },
            "duration": 279,
            "image": [
                {
                    "size": "small",
                    "text": "https://lastfm-img2.akamaized.net/i/u/34s/CENSORED.png"
                },
                {
                    "size": "medium",
                    "text": "https://lastfm-img2.akamaized.net/i/u/64s/CENSORED.png"
                },
                {
                    "size": "large",
                    "text": "https://lastfm-img2.akamaized.net/i/u/174s/CENSORED.png"
                },
                {
                    "size": "extralarge",
                    "text": "https://lastfm-img2.akamaized.net/i/u/300x300/CENSORED.png"
                }
            ],
            "listeners": 1288368,
            "mbid": "CENSORED13ea",
            "name": "Teardrop",
            "rank": 2,
            "url": "https://www.last.fm/music/Massive+Attack/_/Teardrop"
        },
        {
            "artist": {
                "mbid": "CENSORED2711",
                "name": "Radiohead",
                "url": "https://www.last.fm/music/Radiohead"
            },
            "duration": 0,
            "image": [
                {
                    "size": "small",
                    "text": "https://lastfm-img2.akamaized.net/i/u/34s/CENSORED.png"
                },
                {
                    "size": "medium",
                    "text": "https://lastfm-img2.akamaized.net/i/u/64s/CENSORED.png"
                },
                {
                    "size": "large",
                    "text": "https://lastfm-img2.akamaized.net/i/u/174s/CENSORED.png"
                },
                {
                    "size": "extralarge",
                    "text": "https://lastfm-img2.akamaized.net/i/u/300x300/CENSORED.png"
                }
            ],
            "listeners": 1605374,
            "mbid": "CENSORED927f",
            "name": "Karma Police",
            "rank": 3,
            "url": "https://www.last.fm/music/Radiohead/_/Karma+Police"
        },
        {
            "artist": {
                "mbid": "CENSORED2711",
                "name": "Radiohead",
                "url": "https://www.last.fm/music/Radiohead"
            },
            "duration": 239,
            "image": [
                {
                    "size": "small",
                    "text": "https://lastfm-img2.akamaized.net/i/u/34s/CENSORED.png"
                },
                {
                    "size": "medium",
                    "text": "https://lastfm-img2.akamaized.net/i/u/64s/CENSORED.png"
                },
                {
                    "size": "large",
                    "text": "https://lastfm-img2.akamaized.net/i/u/174s/CENSORED.png"
                },
                {
                    "size": "extralarge",
                    "text": "https://lastfm-img2.akamaized.net/i/u/300x300/CENSORED.png"
                }
            ],
            "listeners": 1607794,
            "mbid": "CENSOREDeaa6",
            "name": "Creep",
            "rank": 4,
            "url": "https://www.last.fm/music/Radiohead/_/Creep"
        },
        {
            "artist": {
                "mbid": "CENSORED2711",
                "name": "Radiohead",
                "url": "https://www.last.fm/music/Radiohead"
            },
            "duration": 0,
            "image": [
                {
                    "size": "small",
                    "text": "https://lastfm-img2.akamaized.net/i/u/34s/CENSORED.png"
                },
                {
                    "size": "medium",
                    "text": "https://lastfm-img2.akamaized.net/i/u/64s/CENSORED.png"
                },
                {
                    "size": "large",
                    "text": "https://lastfm-img2.akamaized.net/i/u/174s/CENSORED.png"
                },
                {
                    "size": "extralarge",
                    "text": "https://lastfm-img2.akamaized.net/i/u/300x300/CENSORED.png"
                }
            ],
            "listeners": 215983,
            "mbid": "",
            "name": "Daydreaming",
            "rank": 5,
            "url": "https://www.last.fm/music/Radiohead/_/Daydreaming"
        },
        {
            "artist": {
                "mbid": "CENSOREDef4d",
                "name": "Tame Impala",
                "url": "https://www.last.fm/music/Tame+Impala"
            },
            "duration": 0,
            "image": [
                {
                    "size": "small",
                    "text": "https://lastfm-img2.akamaized.net/i/u/34s/CENSORED.png"
                },
                {
                    "size": "medium",
                    "text": "https://lastfm-img2.akamaized.net/i/u/64s/CENSORED.png"
                },
                {
                    "size": "large",
                    "text": "https://lastfm-img2.akamaized.net/i/u/174s/CENSORED.png"
                },
                {
                    "size": "extralarge",
                    "text": "https://lastfm-img2.akamaized.net/i/u/300x300/CENSORED.png"
                }
            ],
            "listeners": 382761,
            "mbid": "",
            "name": "The Less I Know the Better",
            "rank": 6,
            "url": "https://www.last.fm/music/Tame+Impala/_/The+Less+I+Know+the+Better"
        },
        {
            "artist": {
                "mbid": "CENSOREDb3b5",
                "name": "The White Stripes",
                "url": "https://www.last.fm/music/The+White+Stripes"
            },
            "duration": 449,
            "image": [
                {
                    "size": "small",
                    "text": "https://lastfm-img2.akamaized.net/i/u/34s/CENSORED.png"
                },
                {
                    "size": "medium",
                    "text": "https://lastfm-img2.akamaized.net/i/u/64s/CENSORED.png"
                },
                {
                    "size": "large",
                    "text": "https://lastfm-img2.akamaized.net/i/u/174s/CENSORED.png"
                },
                {
                    "size": "extralarge",
                    "text": "https://lastfm-img2.akamaized.net/i/u/300x300/CENSORED.png"
                }
            ],
            "listeners": 1481075,
            "mbid": "CENSOREDabc5",
            "name": "Seven Nation Army",
            "rank": 7,
            "url": "https://www.last.fm/music/The+White+Stripes/_/Seven+Nation+Army"
        },
        {
            "artist": {
                "mbid": "CENSOREDc4d6",
                "name": "Nirvana",
                "url": "https://www.last.fm/music/Nirvana"
            },
            "duration": 208,
            "image": [
                {
                    "size": "small",
                    "text": "https://lastfm-img2.akamaized.net/i/u/34s/CENSORED.png"
                },
                {
                    "size": "medium",
                    "text": "https://lastfm-img2.akamaized.net/i/u/64s/CENSORED.png"
                },
                {
                    "size": "large",
                    "text": "https://lastfm-img2.akamaized.net/i/u/174s/CENSORED.png"
                },
                {
                    "size": "extralarge",
                    "text": "https://lastfm-img2.akamaized.net/i/u/300x300/CENSORED.png"
                }
            ],
            "listeners": 1789826,
            "mbid": "CENSORED004e",
            "name": "Come as You Are",
            "rank": 8,
            "url": "https://www.last.fm/music/Nirvana/_/Come+as+You+Are"
        },
        {
            "artist": {
                "mbid": "CENSOREDc4d6",
                "name": "Nirvana",
                "url": "https://www.last.fm/music/Nirvana"
            },
            "duration": 301,
            "image": [
                {
                    "size": "small",
                    "text": "https://lastfm-img2.akamaized.net/i/u/34s/CENSORED.png"
                },
                {
                    "size": "medium",
                    "text": "https://lastfm-img2.akamaized.net/i/u/64s/CENSORED.png"
                },
                {
                    "size": "large",
                    "text": "https://lastfm-img2.akamaized.net/i/u/174s/CENSORED.png"
                },
                {
                    "size": "extralarge",
                    "text": "https://lastfm-img2.akamaized.net/i/u/300x300/CENSORED.png"
                }
            ],
            "listeners": 2031860,
            "mbid": "CENSORED4ed7",
            "name": "Smells Like Teen Spirit",
            "rank": 9,
            "url": "https://www.last.fm/music/Nirvana/_/Smells+Like+Teen+Spirit"
        }
    ],
    "limit": 10,
    "page": 1,
    "total": 2525955
}
